app/ui-react/syndesis/src/modules/connections/locales/connections-translations.en.json

```diff
@@ -19,6 +19,17 @@
   "techPreviewPopoverHtml": "<a href='https://access.redhat.com/support/offerings/techpreview' rel='nofollow' target='_blank' role='link'>Technology Previews</a> provide early access to features that are not yet supported. Feedback about these features is welcome. Send a message to <a href='mailto:fuse-online-tech-preview@redhat.com'>fuse-online-tech-preview@redhat.com</a>.",
   "usedByOne": "Used by 1 integration",
   "usedByMulti": "Used by integrations {{count}} times",
-  "unsavedChangesTitle": "Do you really want to cancel?",
-  "unsavedChangesMessage": "You have not finished creating the connection. If you cancel now you will lose data you already entered. Do you still want to cancel?"
+  "create": {
+    "unsavedChangesTitle": "Do you really want to cancel?",
```

Comment:
Maybe this should be in the shared locale since we'd use it more than once across the app? Not a big deal though.

Reply:
Yeah but the description is always different and it felt weird to have two related keys in two different namespaces.